enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts

```diff
@@ -53,3 +57,8 @@ export const getMetabaseInstanceSetupCompleteMessage = (instanceUrl: string) =>
 
   Read more: https://www.metabase.com/docs/latest/installation-and-operation/information-collection
 `;
+
+export const NOT_ENOUGH_TENANCY_COLUMN_ROWS = `
+  At least ${HARDCODED_USERS.length} rows with valid tenancy columns are needed for sandboxing.
+  You can add your tenant's IDs to the "customer_id" user attribute in settings.
+`;
```

Comment:
We need to assign the `customer_id` to our 3 hard-coded users in the Express.js demo, but if we have less than 3 then we don't have enough to demonstrate tenant isolation.